contrib/completion/zsh/_docker

```diff
@@ -1581,8 +1581,7 @@ __docker_plugins() {
 
     # Name
     for line in $lines; do
-        s="${line[${begin[NAME]},${end[NAME]}]%% ##}"
-        s="$s:${(l:7:: :::)${${line[${begin[TAG]},${end[TAG]}]}%% ##}}"
+        s="${${line[${begin[NAME]},${end[NAME]}]%% ##}//:/\:}"
```

Comment:
Instead of using the standard output of `docker plugin ls`, would it perhaps work better to use `--format`, to output the information in a way it's most usable for the completion script?

Reply:
[Lines 1569 to 1580](https://github.com/docker/cli/pull/2903/files#diff-474a9d942d53cc2279d5641e4a9dcfb18958e412ad53940b53423a38033857d6R1569-R1580) could be removed entirely yes. Would displaying NAME be enough (1) or would you prefer having other completion information, like DESCRIPTION (2)? Also should I remove the `:latest` tag like in (2)?

1. ```console
   $ docker plugin inspect [TAB]
   loki:latest         vieux/sshfs:1.3     vieux/sshfs:latest
   ```
2. ```console
   $ docker plugin inspect [TAB]
   loki                          -- Loki Logging Driver
   vieux/sshfs:1.3  vieux/sshfs  -- sshFS plugin for Docker
   ```

Reply:
I went ahead and submitted (2) version.